keras_cv/metrics/object_detection/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 The KerasCV Authors
```

Comment:
Is object_detection the right dir for this? Do we plan to move COCO metrics here?

Reply:
Yeah, I personally beleieve so.  not a user facing API - just an organizational thing.  

Personally I think task is a nice way to divide up metrics